code/modules/surgery/organs/tongue.dm

```diff
@@ -352,19 +352,20 @@
 /obj/item/organ/tongue/tied/handle_speech(datum/source, list/speech_args)
 	var/new_message
 	var/message = speech_args[SPEECH_MESSAGE]
+	var/exclamation_found = findtext(message, "!")
+	var/question_found = findtext(message, "?")
 	var/mob/living/carbon/M = owner
 	new_message = message
 	if(findtext(message, "!"))
-		if(!findtext(message, "?"))
-			M.visible_message("<span class='notice'>[M] raises [M.p_their()] eyebrows.</span>")
-		else
-			M.visible_message("<span class='notice'>[M] lowers one of [M.p_their()] eyebrows, raising the other.</span>")
 		new_message = replacetext(new_message, "!", "")
 
 	if(findtext(message, "?"))
```

Comment:
Betting these were missed?

Reply:
that's the outdated code

Reply:
wait did I actually forget some of the outdated code

Reply:
No, that's not existent code anymore, I got rid of it

Reply:
If we have a variable that stores the results of a find text operation, why not use it here?

Reply:
I apologize, I misunderstood what you were saying before
You're right, I could easily just use that instead, which is what I shall now change

Reply:
No need to apologize, it's not a big deal, just figured ya missed it :).